frontend/src/app/components/SnackbarAlert/SnackbarAlert.js

```diff
@@ -13,6 +13,8 @@ const alertTypeToString = alertType => {
       return 'success'
     case AlertType.ERROR:
       return 'error'
+    default:
```

Comment:
What's the default branch good for?

Reply:
solve warnings of create-react-app scripts. It uses internally some eslint rules when you use `react-scripts start` - extending seems to be complicated - https://create-react-app.dev/docs/setting-up-your-editor/#experimental-extending-the-eslint-config